examples/crate_universe/WORKSPACE.bazel

```diff
@@ -269,6 +269,98 @@ load(
 
 no_cargo_crate_repositories()
 
+###############################################################################
+# U S I N G   C X X   C R A T E
+###############################################################################
+
+# CXX crate is a bit different since there are C++ headers provided.
+
+crates_repository(
+    name = "using_cxx",
+    annotations = {
+        # CXX provides a header file that should be used in C++ sources that depend on cxx.
+        "cxx": [crate.annotation(
+            additive_build_file_content = """
+                # This file is included in the BUILD for the cxx crate, to export its header
+                # file for C++ code to depend on.
+                cc_library(
+                    name = "cxx_cc",
+                    visibility = ["//visibility:public"],
+                    hdrs = ["include/cxx.h"],
+                    srcs = ["src/cxx.cc"],
+                    includes = ["include"],
+                )
+            """,
+            extra_aliased_targets = ["cxx_cc"],
```

Comment:
I think it would be better if we took a dict here, where we map "what the alias in `@using_cxx//:BUILD.bazel` should be called" and map it to "the target in `@using_cxx__cxx-1.0.104//:BUILD.bazel`" it should alias for. It's a nice coincidence that your target here and how you want to expose it are the same, but that's unlikely to always be the case.

(Ideally we'd just add an `additive_build_file_content` for that `@using_cxx` BUILD.bazel file, but we can do that because we don't know the name mangling scheme and the version number, so "a mapping of stuff to push up" seems reasonable).

In your case here, this would be:
```starlark
extra_aliased_targets = {
  "cxx_cc": "cxx_cc",
}
```

but it opens up the ability for you to be more flexible about that mapping, so that someone could create:
```starlark
extra_aliased_targets = {
  "generated_openssl": "openssl_sys_build_script",
}
```

creating "generated_openssl" in the aliases, pointing at "openssl_sys_build_script" in the generated per-crate repo.

or something along those lines.

Reply:
Done.